CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 0.9.0
+
+This release adds support for Linux ARM (armv7l and armv8l 32 bit mode).
```

Comment:
Similar to scie jump, if you check all the RELEASE.md prep boxes, I can get a release out. You're missing a corresponding version bump for the package.

Reply:
Super. I'll fix.

Reply:
I get the following warning:

```bash
nox > mypy --python-version 3.12 science tests test-support noxfile.py docs
science/commands/complete.py:19: error: Incompatible types in assignment (expression has type "Process | None", variable has type "Process")  [assignment]
Found 1 error in 1 file (checked 45 source files)
nox > Command mypy --python-version 3.12 science tests test-support noxfile.py docs failed with exit code 1
nox > Session check-/home/lauszus/Github/lift/.nox/PBS/20241206/3.12.8/install_only_stripped/python/bin/python3.12 failed.
nox > Ran multiple sessions:
nox > * fmt-/home/lauszus/Github/lift/.nox/PBS/20241206/3.12.8/install_only_stripped/python/bin/python3.12: success
nox > * lint-/home/lauszus/Github/lift/.nox/PBS/20241206/3.12.8/install_only_stripped/python/bin/python3.12: success
nox > * check-/home/lauszus/Github/lift/.nox/PBS/20241206/3.12.8/install_only_stripped/python/bin/python3.12: failed
```

But I ignored it, as I haven't touched that file or made any changes to the `psutil` dependency.

Reply:
> Aha, not so easy. There will need to be some test mods for armv7l. I'll check back in on your progress in the morning Nevada, US time.

Super. I'll try to fix the pytest tests. Seems like it is running it for armv7l even though it is not available.